### PR TITLE
[M3-lab-supervised-classification] Fernando Gamiño

### PR DESCRIPTION
**Pull request by FernandoGaminho**

### CODE REVIEW

**Comment by ta-data-mex** [comment]
Buen trabajo Fer 👍  es muy útil crear funciones para no repetir código, solo creo que se te paso algo 🙈  aquí:
def allModels_testing(lista_modelos, data):
    X_train, X_test, y_train, y_test = train_test_split(data.loc[:, data.columns != 'e'], data['e'])
    
    for i in lista_modelos:
        modelo = i()
        modelo.fit(X_train, y_train)
        train_score = modelo.score(X_train, y_train)
        test_score = modelo.score(X_train, y_train) <----- esto esta incorrecto
        test_score = modelo.score(X_test, y_test)
 
        print('MODELO:  ',str(i).split(sep = '.')[-1])
        print('Train score:',train_score)
        print('Test score:',test_score,'\n')

y por alguna razón la función no función porque no te trae los resultados esperados.

Cuando tu aplicas modelo.score la métrica que te trae es mean accuracy pero recuerda que hay otras métricas para evaluar modelo de clasificación, como: Recall, Precision, F1- Score y la famosísima matriz de confusión que nos ayuda a ver la calidad del modelo de manera visual.
